### PR TITLE
fix: upgrade esri js to fix map click bug in chrome

### PR DESCRIPTION
**Pull request by stdavis**
Ref: https://support.esri.com/en/bugs/nimbus/QlVHLTAwMDEzODY4NA==

I haven't tested this. This is just what I had to do for similar projects.

### CODE REVIEW

File: PLSS/profiles/build.profile.js [hunk 1]
```diff
@@ -20,7 +20,9 @@ var profile = {
                 'dojox/gfx/svg',
                 'dojox/gfx/svgext',
                 'dojox/gfx/shape',
-                'esri/dijit/Attribution'
+                'esri/dijit/Attribution',
+                'esri/layers/VectorTileLayerImpl',
+                'esri/layers/vectorTiles/views/vectorTiles/WorkerTileHandler'
```
Comment (steveoh) [inline PLSS/profiles/build.profile.js:25]: **question**: didn't this break the build?

File: PLSS/src/app/run.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ((function () {
     var config = {
-        baseUrl: '/src',
+        baseUrl: '/.',
```
Comment (steveoh) [inline PLSS/src/app/run.js:3]: **question**: this wasn't used in the version deployed correct?

Comment (stdavis) [reply PLSS/src/app/run.js:3]: Yes, I believe that it was.

File: PLSS/profiles/build.profile.js (hunk 1 above)
Comment (stdavis) [reply PLSS/profiles/build.profile.js:25]: No, `esri/layers/vectorTiles/core/workers/worker` was the one causing issues.

**Pull request merged**